### ISSUE TITLE
question: How to Install

### ISSUE THREAD

**Issue by gristim** [Baldomo/open-in-mpv#30]
### Question Description

I am currently on Linux and it is unclear to me how to install the program correctly. 
I have done the following:
- Install the Firefox Browser Extension
- Download the Linux.tar and run `install-protocol.sh`

It is not immediately obvious what the next steps are, but clicking on the browser extension does nothing, regardless of whether mpv is already open or not. 

Thanks for your help!

### Checklist:

- [x] I have checked for existing issues that describe my questions prior to opening this one.
- [X] I understand that improperly formatted questions may be closed without explanation.

**Comment by Baldomo** [comment]
Hello! The recommended way of installing the program is using the Makefile but for the binary release, you can just copy what `make` does in the target `install`: 

https://github.com/Baldomo/open-in-mpv/blob/d599a6470c22a5a980e5da4894793bc8f8ae9c91/Makefile#L48-L49

tldr: just copy the binary to somewhere that's in your PATH
